src/App.jsx

```diff
@@ -1,104 +1,28 @@
+/* eslint-disable */
+
 import './App.scss';
+import commetsFromServer from './api/comments.json'
```

Comment:
There is a typo in the variable name `commetsFromServer`. It should be `commentsFromServer` to match the file name `comments.json`.